scipy/optimize/_optimize.py

```diff
@@ -2248,12 +2248,19 @@ def get_bracket_info(self):
             if (xa > xc):  # swap so xa < xc can be assumed
                 xc, xa = xa, xc
             if not ((xa < xb) and (xb < xc)):
-                raise ValueError("Not a bracketing interval.")
+                 raise ValueError(
+                    "Bracketing values (xa, xb, xc) do not"
+                    "fulfill this requirement: (xa < xb) and (xb < xc)"
```

Comment:
My original suggestion was correct. We need the space because the strings are going to be concatenated without adding any spacing.
```suggestion
                    " fulfill this requirement: (xa < xb) and (xb < xc)"
```